src/shared/Microsoft.Git.CredentialManager/Application.cs

```diff
@@ -130,6 +130,11 @@ private bool WriteException(Exception ex)
                     break;
             }
 
+            // If tracing is enabled then also print the stack trace to stderr
+            bool printStack = Context.Settings.GetTracingEnabled(out _);
+            if (printStack)
+                Context.Streams.Error.WriteLine(ex.StackTrace);
+
```

Comment:
hyper-nit: the variable `printStack` is not necessary as it is not re-used.

Reply:
Yeah, I'm explicitly putting this intermediate variable there to make it very clear the reason for using `GetTracingEnabled(..)` in "normal" exception logging code. I guess the comment above also explains this explicit decision.